snuba/web/rpc/v1/endpoint_time_series.py

```diff
@@ -105,4 +111,12 @@ def _execute(self, in_msg: TimeSeriesRequest) -> TimeSeriesResponse:
                 "This endpoint requires meta.trace_item_type to be set (are you requesting spans? logs?)"
             )
         resolver = self.get_resolver(in_msg.meta.trace_item_type)
-        return resolver.resolve(in_msg)
+        try:
+            return resolver.resolve(in_msg)
+        except QueryException as e:
+            if e.extra[
+                "code"
+            ] == 241 or "DB::Exception: Memory limit (for query) exceeded" in str(e):
+                metrics.increment("endpoint_trace_item_table_OOM")
```

Comment:
You should instead use the `metrics` property included in the class

https://github.com/getsentry/snuba/blob/master/snuba/web/rpc/__init__.py#L105-L111

`self.metrics.increment("oom_query")`